### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by casqan** [B-Team-Organisation/Fantasy-Chess#5] [Announcements]
Information on progress as well as Documentation.

**Comment by Jacinto-S** [comment]
# Story Point Reference
![image](https://github.com/user-attachments/assets/84b82cb2-ccf8-4219-a20c-8c6b5ee0d47a)

**Comment by Jacinto-S** [comment]
# General Project Roadmap
![image](https://github.com/user-attachments/assets/a12267db-0be6-4c93-919c-3d68c27940d5)
### Heute = 22.10
![image](https://github.com/user-attachments/assets/e928f7df-16aa-4d97-b2bc-b7f13751ee98)

**Comment by Jacinto-S** [comment]
# Bootstrapping Results
![image](https://github.com/user-attachments/assets/20aac551-1b6a-442e-92ea-ebbc7a8b75f6)
![image](https://github.com/user-attachments/assets/f3f03340-88b3-4cdf-a46a-64a052e72f2e)
![image](https://github.com/user-attachments/assets/da527400-e596-44a6-87b8-64df0325eb0c)

**Comment by Jacinto-S** [comment]
# Sequenzdiagram für den generellen Ablauf
![Sequence Diagram](https://www.plantuml.com/plantuml/png/fPD1Rjim44NtFiN0-roWA8g4WPiMpQA8YtPDMP_MeDJ884S9UfSkVOQkijF5Ijc6eHWgeSfUU4E-Dv-Vr6rK6xIj_M9vxncYTBUacm-q_84Y8ZmZBEaTFNRxKcbptF3Sr7HU7MBlRzvYbye7MoEDKz3wRFwEKB2KK8gygQXcfH8rhKVauu9LgBgzLtfiYk8NVHRNXDegDmAPC3_XtH_d14fWmSbYMpUMminhncIQCTSsOwvh9ZF_Ms3jHmPgd4E2Jg82l1W1acPGTvfUxrotbwrIt34ymcM2hC8C3s30seU9P1BHxPMthModtEvQRL2KjEuZJ4czng7n-ye6-rFdXCuR9eHE02jRZu7ZQnCvVtFJ1PDRE4DnNWiPW_LANy123tXX4GHQYSPDnIZJweBIRq9H1WwLZvwZuWrZfZDZk_x5VknUgv3iU47_06DnS85i6sUgxcGRsBbhJDtl0c4NkbUdr1G4bkDhH-ZBRKGWzCdgiTIS_DptuCjxQgLC_1DYwnCm0JSvV04VGQj1nCegRKynCmbpztbyQGVX_ntl1Cq-tGKDAHUtwQ-j_Ly0)

**Comment by WalkerLukas** [comment]
# Sequenzdiagram für den generellen Ablauf
![Sequence Diagram](https://www.plantuml.com/plantuml/png/fPH1Rjim44NtFiN0so8NA8eYOU2cGRCeug9TKzQND2WrDCXHZDwbIv-Xg-nqiKAsGDk882ZMmh04pZpD__paRLGRjAdTOddl6A9qjwIRpxJyXImYl20iwICzjVjSQNCiUF4rfTNpjaytbsrfpLo7D2F8Vie7MoEgLT0waNyYP2mvb29lAgfPAKTDwpVmwm3E-osjjmo7G3ll78JYuLKMH4GJ9lmIRAdqvBFi3qlXGssLlK38NYBCF4JQbc74AbJjtlKGUZnHfWjPXVPVKGYKm8AUP9lYtAzBGVEc6VVaHs1LoX785mK4qiCoS8SOQkfGIMOtHRqjshsKsczf69Xa76FEnHYmTaD5CeZQxfNBHigfxgxL1aL-3DjK1MiSadjVsM1_wxK2nYbJ1JmAArk7Cyvt7ta0OkP0JYUIE2g9trGalAcK81et5IFlAz_bOVfC750_nSVffFGmmLI30fMBZgFYJP-PtZScOV36QL-he6ImGnVYc1ru8Dl46RWxAGCNnLnQ-pT3s8RsjL3o6O4bNT4Fs7aH27srkaXiwvI-WQLRPX44Mfsoaiec1YQTz-T6SWWz8pPEhRu3OyOmYGCNGQipSPLLAdk1YvIaAmmqE2Rr09uf_yEHCiE6QJX8lg3-l_lTGlAcTly1)

**Comment by bodoai** [comment]
Fotos vom Rapid Prototyping Event am 22.10.2024:

![IMG_20241022_114146](https://github.com/user-attachments/assets/dbd88833-80ed-44ee-9dd2-d60d16a874c0)
![IMG_20241022_114139](https://github.com/user-attachments/assets/9669d1d1-61a8-4edc-9fbe-47320cdf64bf)
![IMG_20241022_113748](https://github.com/user-attachments/assets/9f26cbc3-539e-47ee-bb8f-4761e526e8b0)
